### PR TITLE
Adds OSS::UrlInput component + tests & documentation

### PR DESCRIPTION
**Pull request by Miexil**
### What does this PR do?

Adds OSS::UrlInput component + tests & documentation

Related to: #https://github.com/upfluence/backlog/issues/2016

### Good PR checklist
- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [x] Added/updated tests
- [x] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

**Review by phndiaye — APPROVED** [review]

File: addon/components/o-s-s/url-input.stories.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      },
+      control: { type: 'text' }
+    },
+    errorMessage: {
+      description: 'An error message that will be displayed below the input-group.',
+      table: {
+        type: {
+          summary: 'string'
+        },
+        defaultValue: { summary: false }
```
Comment (JulienVannier66) [inline addon/components/o-s-s/url-input.stories.js:44]:
```suggestion
        defaultValue: { summary: undefined }
```
It is a string value

File: addon/components/o-s-s/url-input.stories.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+export const BasicUsage = DefaultUsageTemplate.bind({});
+BasicUsage.args = {
+  value: '',
+  prefix: 'https://',
+  suffix: '@shopify.com',
+  placeholder: 'subdomain',
+  disbaled: 'false',
+  errorMessage: 'Not a valid subdomain',
+  validationRegex: /^[a-zA-Z0-9]+[a-zA-Z0-9-._]*[a-zA-Z0-9]+$/,
+  onChange: action('')
```
Comment (JulienVannier66) [inline addon/components/o-s-s/url-input.stories.js:117]:
```suggestion
  onChange: action('onChange')
```
To have a better display in the doc action section

File: addon/components/o-s-s/url-input.stories.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        defaultValue: { summary: false }
+      },
+      control: { type: 'text' }
+    },
+    onChange: {
+      type: { name: 'Method' },
+      description: 'Method called every time the input is updated',
+      table: {
+        type: {
+          summary: 'onChange(value: string): void'
```
Comment (JulienVannier66) [inline addon/components/o-s-s/url-input.stories.js:53]:
```suggestion
          summary: 'onChange(value: string, isValid: boolean): void'
```
It is the declaration in the ts file

File: tests/integration/components/o-s-s/url-input-test.ts [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+import typeIn from '@ember/test-helpers/dom/type-in';
+import sinon from 'sinon';
+
+module('Integration | Component | o-s-s/url-input', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(hbs`<OSS::UrlInput @prefix="https://" />`);
+
+    assert.true(true);
```
Comment (JulienVannier66) [inline tests/integration/components/o-s-s/url-input-test.ts:15]: 🙈

**Review by JulienVannier66 — APPROVED** [review]
Nice work!

**Pull request merged**